parent/pom.xml

```diff
@@ -1006,6 +1007,10 @@
               </property>
             </properties>
             <reportNameSuffix>${env.SUREFIRE_REPORT_NAME_SUFFIX}</reportNameSuffix>
+            <!--
+              ensure junit5 display names are properly reported; see the documentation for more
+              https://maven.apache.org/surefire/maven-surefire-plugin/examples/junit-platform.html
+              -->
```

Comment:
🙃 
```suggestion
            <!--
              ensure junit5 display names are properly reported; see the documentation for more
              https://maven.apache.org/surefire/maven-failsafe-plugin/examples/junit-platform.html
              -->
```

Reply:
Fair enough :smile: They're essentially the same plugin :wink: If you look at the surefire repo you'll find failsafe nested in it and inheriting a lot of stuff